config/zones/US-MIDA-PJM.yaml

```diff
@@ -80,9 +89,12 @@ emissionFactors:
         source: Electricity Maps, 2021 average
         value: 372.3153209954643
     oil:
-      datetime: '2021-01-01'
-      source: eGrid 2020
-      value: 751.5213327
+      - datetime: '2020-01-01'
+        source: eGrid 2020
+        value: 751.5213327
+      - datetime: '2021-01-01'
+        source: eGRID 2021
+        value: 970.02
```

Comment:
![image](https://github.com/electricitymaps/electricitymaps-contrib/assets/107848894/6f60daaa-5837-4ea3-91c9-189d01de2b2a)
We have a big increased compared to 2021 but it is still close to the average value